jerry-core/parser/js/lexer.h

```diff
@@ -171,7 +171,7 @@ typedef struct
 
 void lexer_init (const jerry_api_char_t *, size_t, bool);
 
-token lexer_next_token (void);
+token lexer_next_token (bool = false);
```

Comment:
Could you please remove default argument value and use explicit argument value at every call site. We've agreed to use C++ features as little as possible in Jerry.